lib/plottr_components/src/components/dashboard/account/proOnboarding/ProStep1.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react'
+import PropTypes from 'react-proptypes'
+import { t } from 'plottr_locales'
+import OnboardingStep from '../../../onboarding/OnboardingStep'
+import { StepBody, StepHeader } from '../../../onboarding/Step'
+import UnconnectedFirebaseLogin from '../../../FirebaseLogin'
+
+const ProStep1Connector = (connector) => {
+  const {
+    platform: { useSettingsInfo },
+  } = connector
+
+  const FirebaseLogin = UnconnectedFirebaseLogin(connector)
+
+  const ProStep1 = ({ nextStep }) => {
+    const [settings] = useSettingsInfo()
+    useEffect(() => {
+      if (settings.user.id) nextStep()
+    }, [settings])
```

Comment:
Is `settings` enough here?
I think that the object is mutable, so the hook won't see the setting change.